Makefile

```diff
@@ -117,7 +117,9 @@ OBJECTS=$(patsubst $(SRCDIR)/%.c,$(BINDIR)/%.o,$(SOURCES))
 CC_DEPS = $(patsubst $(SRCDIR)/%.c, $(BINDIR)/%.d, $(SOURCES))
 
 CC_FLAGS += \
-	-fPIC -std=gnu99 \
+	-fPIC \
+	-fcommon \
```

Comment:
IMO this is more risky than my change, but I'll go with it.

Reply:
@rafie can you explain why? What risks do you see with it?